core/iwasm/interpreter/wasm.h

```diff
@@ -480,18 +480,22 @@ typedef void *WASMString;
 #endif /* end of WASM_ENABLE_STRINGREF != 0 */
 #endif /* end of WASM_ENABLE_GC != 0 */
 
-typedef struct WASMTable {
+typedef struct WASMTableType {
     uint8 elem_type;
     /**
      * 0: no max size and not shared
-     * 1: hax max size
+     * 1: has max size
      * 2: shared
      */
     uint8 flags;
     bool possible_grow;
     uint32 init_size;
     /* specified if (flags & 1), else it is 0x10000 */
     uint32 max_size;
+} WASMTableType;
```

Comment:
Had better move field elem_ref_type from WASMTable/WASMTableImport into WASMTableType, it is also one of the basic type info:
```C
#if WASM_ENABLE_GC != 0
    WASMRefType *elem_ref_type;
#endif
```